host/vite.config.ts

```diff
@@ -1,7 +1,22 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react-swc'
+import federation from '@originjs/vite-plugin-federation'
```

Comment:
Did you add federation in this project? Impressive.

Reply:
Yes, federation is included.